### PR TITLE
[JS-loaders] Stabilize feature

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Stabilze JS-loader functions
# Why

GA's the feature for GroupMe

# How Tested

Built and tested locally CI to validate

### CODE REVIEW

**Review by JamesBurnside — APPROVED** [review]

**Review by PorterNan — APPROVED** [review]

**Review by mgamis-msft — APPROVED** [review]

File: common/config/babel/features.js [hunk 1]
```diff
@@ -97,6 +97,8 @@ module.exports = {
     // Feature for showing notifications
     "notifications",
     // Feature for tracking beta start call identifier
-    'start-call-beta'
+    'start-call-beta',
+    // feature for adding JS helpers to the UI library API
+    "composite-js-helpers",
```
Comment (JamesBurnside) [inline common/config/babel/features.js:102]: also remove this from the beta list

File: packages/acs-ui-javascript-loaders/src/callCompositeLoader.ts [hunk 2]
```diff
@@ -22,41 +22,41 @@ import { initializeIcons } from '@fluentui/react';
 
 /**
  * Props for the OutboundCallComposite that you can use in your application.
```
Comment (JamesBurnside) [inline packages/acs-ui-javascript-loaders/src/callCompositeLoader.ts:24]: Update this comment for each composite type, its generic and doesn't say anything the name already tells the developer. Here is a good place to explain why there are two options bag and `@link` to where they can read more about each

File: packages/acs-ui-javascript-loaders/src/callCompositeLoader.ts [hunk 3]
```diff
@@ -22,41 +22,41 @@ import { initializeIcons } from '@fluentui/react';
...
   token: string;
   displayName: string;
   locator: CallAdapterLocator;
-  options?: AzureCommunicationCallAdapterOptions;
+  adapterOptions?: AzureCommunicationCallAdapterOptions;
+  callCompositeOptions?: CallCompositeOptions;
 };
 
 /**
  * Loader function for the OutboundCallComposite that you can use in your application.
```
Comment (JamesBurnside) [inline packages/acs-ui-javascript-loaders/src/callCompositeLoader.ts:37]: `@link` OutboundCallComposite so the developer can read more

**Review by JamesBurnside — CHANGES_REQUESTED** [review]
marking as request changes for the update to features file

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**